### PR TITLE
copy edit: /help

### PR DESCRIPTION
**Pull request by kwalrath**
Some notes:

* I'd like to use `<dl>`s for some of the lists, but didn't because I wasn't sure how it'd be formatted.
* We could be more specific in many places. For example:
  * What is searched? The package source code, the pubspec, what you can actually see on pub.dartlang.org, ...?
  * Do we support combining multiple search expressions?
* We should show some examples of searches.

### CODE REVIEW

**Comment by kevmoo** [comment]
@kwalrath – we just made your life hard. @isoos just moved everything to new directories.

Should just be a copy-paste when you rebase

**Comment by kwalrath** [comment]
Contributes to #745.

**Comment by isoos** [comment]
> Do we support combining multiple search expressions?

We do.

File: app/views/help.mustache [hunk 1]
```diff
@@ -5,35 +5,47 @@
...
+    from scores for the package's <a href="#popularity">popularity</a>,
+    <a href="#health">health</a>, and <a href="#maintenance">maintenance</a>.
   </p>
 
   <h3 id="popularity">Popularity</h3>
   <p>
-    Popularity score is good proxy for a package's use, derived from download statistics.
-    It is based on actual download counts, but compensated for automated tools (like
-    continuous builds fetching the package on each change request).
+    The popularity score—representing how often a package is used—is derived
```
Comment (isoos) [inline app/views/help.mustache]: `used—is` - this looks strange

**Comment by isoos** [comment]
@kwalrath: thanks, this is really appreciated!

**Comment by kevmoo** [comment]
@kwalrath - please squash and merge!

File: app/views/help.mustache (hunk 1 above)
Comment (kwalrath) [reply app/views/help.mustache]: I'll check the appearance once it's up... it might not be so odd once it's displayed using better fonts.

**Pull request merged**